common/packager.py

```diff
@@ -92,7 +92,7 @@ def package_installers(working_dir, build_platforms):
       
   # The directory which holds the files to be added to the bundles, specified
   # both relative to the working directory and with an absolute path.
-  config_dir_rel = os.path.join('.', constants.TEMP_DIR_NAMES['config'])
+  config_dir_rel = constants.TEMP_DIR_NAMES['config']
```

Comment:
Seems to fix the addressed issue. Unpacking `seattle_mac.tgz` or `seattle_linux.tgz` results in one directory named `seattle` containing amongst others `seattle_repy/vesselinfo`.

As for `seattle_win.zip`, when `unzip`ing it results in two directories: `seattle` containing only `seattle_repy/vessleinfo` and `seattle_win` containing `seattle` and `settle_repy` without vesselinfo.